Ginger/Ginger/Repository/VariableRepositoryPage.xaml.cs

```diff
@@ -149,7 +149,7 @@ private void ViewRepositoryItemUsage(object sender, RoutedEventArgs e)
 
         private void grdVariables_PreviewDragItem(object sender, EventArgs e)
         {
-            if (DragDrop2.DragInfo.DataIsAssignableToType(typeof(VariableBase)))
+            if (DragDrop2.mDragInfo.DataIsAssignableToType(typeof(VariableBase)))
             {
```

Comment:
so for now only Actions supports multi items drop? or also Variabels/Activities?

Reply:
Only Actions and Activities(from SharedRepository) for now. Shall I update for the ListView(s) too?

Reply:
yes, we need to allow multiple items D&D also from ListView to ListView (like in case of adding Variabels)

Reply:
Done !